frontend-service/src/pages/landingPage.tsx

```diff
@@ -19,15 +21,25 @@ const LandingPage = () => {
         top={'top-8'}
         handleReset={handleReset}
         formData={[
-          { label: 'Name', className: 'col-span-1 flex flex-col', type: 'text' },
-          { label: 'Age', className: 'col-span-1 flex flex-col', type: 'number' },
-          { label: 'Contact', className: 'col-span-1 flex flex-col', type: 'text' },
-          { label: 'City', className: 'col-span-1 flex flex-col', type: 'text' },
-          { label: 'Speciality', className: 'col-span-2 flex flex-col', type: 'text' },
-          { label: 'Hospital', className: 'col-span-2 flex flex-col', type: 'text' },
-          { label: 'Doctor', className: 'col-span-1 flex flex-col', type: 'text' },
-          { label: 'Appointment Date', className: 'col-span-1 flex flex-col', type: 'text' }
+          { label: 'Name', name: 'name', className: 'col-span-1 flex flex-col', type: 'text' },
+          { label: 'Age', name: 'age', className: 'col-span-1 flex flex-col', type: 'number' },
+          { label: 'Contact', name: 'contact', className: 'col-span-1 flex flex-col', type: 'text' },
+          { label: 'City', name: 'city', className: 'col-span-1 flex flex-col', type: 'text' },
+          { label: 'Speciality', name: 'speciality', className: 'col-span-2 flex flex-col', type: 'text' },
+          { label: 'Hospital', name: 'hospital', className: 'col-span-2 flex flex-col', type: 'text' },
+          { label: 'Doctor', name: 'doctor', className: 'col-span-1 flex flex-col', type: 'text' },
+          { label: 'Appointment Date', name: 'appDate', className: 'col-span-1 flex flex-col', type: 'date' }
         ]}
+        validationSchema={ object({
```

Comment:
Create a folder named validation and store the validation in that folder component wise